test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -2690,18 +2744,20 @@ def forward(self, input):
     @disableScriptTest()
     def test_split_size_as_list(self):
         class SplitModel(torch.nn.Module):
-            def forward(self, input):
+            def forward(self, input, split_sizes: List[int]):
```

Comment:
is this updated to pass scripting? this test is still disabled for script test though

Reply:
Yes, this surfaces another known issue which is related to .append when scripting.